src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
+     * @author madhankumar
+     */
+    @Setter
+    @Getter
+    @ToString
+    @Data
+    @Builder
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public class ChatRequest {
```

Comment:
:warning: The class has multiple annotations (@Setter, @Getter, @ToString, @Data, @Builder, @NoArgsConstructor, @AllArgsConstructor) which can lead to excessive code generation and violate the Single Responsibility Principle (SRP). (Severity: Major)